### PR TITLE
pubsub: remove all batching knobs (TopicOptions, SubscriptionOptions)

### LINKED ISSUE #677: pubsub: remove all batching knobs (TopicOptions, SubscriptionOptions)
As per @rsc's suggestion, we can dynamically vary the batch size, making it smaller when messages are less frequent, and vice versa. Then users don't have to fiddle with knobs and wind up with suboptimal settings.

### PR DESCRIPTION
**Pull request by ijt**
Fixes #677

### CODE REVIEW

File: internal/pubsub/acks_test.go [hunk 1]
```diff
@@ -124,17 +121,22 @@ func TestMultipleAcksCanGoIntoASingleBatch(t *testing.T) {
 func TestTooManyAcksForASingleBatchGoIntoMultipleBatches(t *testing.T) {
 	ctx := context.Background()
 	var sentAckBatches [][]driver.AckID
-	ids := []int{rand.Int(), rand.Int()}
+	// This value of n is chosen large enough that it should create more
+	// than one batch. Admittedly, there is currently no explicit guarantee
```
[the remark this answers is not in the document]
Comment (ijt) [reply internal/pubsub/acks_test.go]: Ack.

**Comment by ijt** [comment]
The build is failing again for Wire on Windows. Can we shut off Wire builds in PRs like this?

**Comment by vangent** [comment]
> Can we shut off Wire builds in PRs like this?

Just restart the Travis job for now. It's annoying, but this problem will go away soon when we move Wire to its own repo.

**Comment by ijt** [comment]
Okay.

**Comment by ijt** [comment]
The Windows build flaked again. Could we temporarily disable that to unblock our work?

**Pull request merged**